modules/examiner/jsx/editExaminer.js

```diff
@@ -0,0 +1,355 @@
+import Panel from 'Panel';
+import Loader from 'Loader';
+
+/**
+ * Examiner Module
+ *
+ * Edit examiner page to the module that allows changes to the examiner's
+ * certification status for enabled instruments
+ *
+ * Renders the Edit Examiner page consisting of FormElement
+ * and Panel components.
+ *
+ * @author Zaliqa Rosli
+ * @version 1.0.0
+ */
+class EditExaminer extends React.Component {
+
+  constructor(props) {
+    super(props);
+
+    this.state = {
+      isLoaded: false,
+      data: {},
+      formData: {},
+      statusOptions: {
+        null: "N/A",
+        notCertified: "Not Certified",
+        inTraining: "In Training",
+        certified: "Certified"
+      }
+    };
+
+    /**
+     * Bind component instance to custom methods
+     */
+    this.fetchData = this.fetchData.bind(this);
+    this.setFormData = this.setFormData.bind(this);
+    this.handleSubmit = this.handleSubmit.bind(this);
+    this.editForm = this.editForm.bind(this);
+    this.changeLog = this.changeLog.bind(this);  
+  }
+
+  componentDidMount() {
+    this.fetchData();
+  }
+
+  /**
+   * Retrieve data from the provided URL and save it in state
+   */
+  fetchData() {
+    const self = this;
```

Comment:
I don't believe there's a need for this line since this should be bound to the ajax call success clause via the arrow function.
```suggestion
```